client/src/components/application/Transactions.vue

```diff
@@ -19,25 +19,25 @@
             </v-flex>
           </v-layout>
 
-          <div class="transaction-header-names">
+          <div id="transaction-header-names">
```

Comment:
There's a single instance of this, and changing it to an id is consistent with the other blocks.